.github/workflows/ci.yaml

```diff
@@ -17,14 +17,26 @@ jobs:
         - '3.0'
         - '3.1'
         - '3.2'
+        - '3.3'
+        - '3.4'
         bundler-version:
-        - '2.1'
-        - '2.2'
-        - '2.3'
         - '2.4'
-        include:
-        - ruby-version: '2.6'
-          bundler-version: '2.0'
+        - '2.5'
+        - '2.6'
+        - '2.7'
+        exclude:
+        - ruby-version: '2.7'
+          bundler-version: '2.5'
+        - ruby-version: '2.7'
+          bundler-version: '2.6'
+        - ruby-version: '3.0'
+          bundler-version: '2.6'
+        - ruby-version: '2.7'
+          bundler-version: '2.7'
+        - ruby-version: '3.0'
+          bundler-version: '2.7'
+        - ruby-version: '3.1'
+          bundler-version: '2.7'
```

Comment:
The exclude list was easier to follow than an include list.  As we change the matrix, we can certainly go back to an include list.